src/main/java/com/nirima/jenkins/plugins/docker/builder/DockerBuilderPublisher.java

```diff
@@ -21,9 +21,7 @@
 import hudson.Extension;
 import hudson.FilePath;
 import hudson.Launcher;
-import hudson.model.AbstractProject;
-import hudson.model.Item;
-import hudson.model.TaskListener;
+import hudson.model.*;
```

Comment:
Please undo these changes to imports (it helps to keep code churn to a minimum).
(I guess this was your IDE "tidying things up" - it seems that no two people's IDEs have the same settings :wink:)